spec/models/saved_recipie_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+### ERROR: Validation failed: Email has already been taken
+## Everytime test is run email can no longer be used until database is reset
+
+RSpec.describe SavedRecipie, type: :model do
+  describe 'relationships' do 
+    it { should.belong_to(:user) }
+    it { should.belong_to(:recipie) }
```

Comment:
We dont have a Recipie Model or DB table since that is an API? How is this test working? Are you thinking we need a Recipie model? Is there a way to create this relationship with out the Model/DB?